libraries/WiFiClientSecure/src/WiFiClientSecure.cpp

```diff
@@ -254,9 +282,10 @@ int WiFiClientSecure::read(uint8_t *buf, size_t size)
         buf++;
         peeked = 1;
     }
-    
-    int res = get_ssl_receive(sslclient, buf, size);
+        res = get_ssl_receive(sslclient, buf, size);
+
     if (res < 0) {
+        log_d("Closing connection on failed read");
```

Comment:
Change debug to error message to match same log level of message on line 242.
```suggestion
        log_e("Closing connection on failed read");
```